df/plugin.go

```diff
@@ -84,6 +84,12 @@ var (
 		"securityfs",
 		"devpts",
 		"mqueue",
+		"hugetlbfs",
+		"nsfs",
```

Comment:
I think that it could be a good idea to have this entire list as a configurable option (passed within the task) in the short/mid-term future ... but this will require more work